main.css

```diff
@@ -933,3 +479,16 @@ hr {
   margin-top: 0;
   margin-bottom: 10px;
 }
+
+/* This is for the nav tag */
+@media (max-width: 768px) {
+  .hamburger-icon {
+    display: inline;
+    color: #6070ff;
+  }
```

Comment:
- [ ] The Desktop version is still showing the mobile version you did earlier, kindly, make sure that it follows this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Reply:
As a hint, you will need these breakpoints: `@media screen and (min-width: 1080px) {}` and `@media screen and (min-width: 750px) {}`